src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java

```diff
@@ -5,6 +5,10 @@
 import spark.Request;
 
 public class RequestUtil {
+    private RequestUtil() {
+        // utility class
+    }
```

Comment:
Usually documentation for private members isn't generated anyway and it's justified by the comment inside?